beacon_node/eth2_libp2p/src/peer_manager/peerdb/peer_info.rs

```diff
@@ -101,13 +101,59 @@ impl<T: EthSpec> PeerInfo<T> {
         false
     }
 
+    /// Obtains the client of the peer.
+    pub fn client(&self) -> &Client {
+        &self.client
+    }
+
+    /// Returns the listening addresses of the Peer.
+    pub fn listening_addresses(&self) -> &Vec<Multiaddr> {
+        &self.listening_addresses
+    }
+
+    /// Returns the connection direction for the peer.
+    pub fn connection_direction(&self) -> Option<&ConnectionDirection> {
+        self.connection_direction.as_ref()
+    }
+
+    /// Returns the sync status of the peer.
+    pub fn sync_status(&self) -> &SyncStatus {
+        &self.sync_status
+    }
+
+    /// Returns the metadata for the peer if currently known.
+    pub fn meta_data(&self) -> Option<&MetaData<T>> {
+        self.meta_data.as_ref()
+    }
+
+    /// Returns whether the peer is a trusted peer or not.
+    pub fn is_trusted(&self) -> bool {
+        self.is_trusted
+    }
+
+    /// The time a peer is expected to be useful until for an attached validator. If this is set to
+    /// None, the peer is not required for any upcoming duty.
+    pub fn min_ttl(&self) -> Option<&Instant> {
+        self.min_ttl.as_ref()
+    }
+
+    /// The ENR of the peer if it is known.
+    pub fn enr(&self) -> Option<&Enr> {
+        self.enr.as_ref()
+    }
+
     /// Returns if the peer is subscribed to a given `Subnet` from the gossipsub subscriptions.
     pub fn on_subnet_gossipsub(&self, subnet: &Subnet) -> bool {
         self.subnets.contains(subnet)
     }
 
     /// Returns the seen IP addresses of the peer.
```

Comment:
Probably should update docs here
```suggestion
    /// Returns the seen addresses of the peer.
```